src/em_keyboard/__init__.py

```diff
@@ -157,13 +151,12 @@ def cli() -> None:
             sys.exit(1)
 
     # Process the results.
-    results = (translate(lookup, name) for name in names)
-    results = list(itertools.chain.from_iterable(results))
+    results = tuple(translate(lookup, name) for name in names)
 
     if None in results:
         no_copy = True
         missing = True
-        results = (r for r in results if r)
+        results = tuple(r for r in results if r)
```

Comment:
This currently works as a generator too, but seems a bit dangerous. The generator is used twice in the code that follows, so on the second time (i.e. `results = "".join(results)`) it will be empty. Luckily in that case the value `results` is always unused. So it works, but just barely ;)